### DISCUSSION TITLE
If you can offer more details about how to deploy or develop the project locally?

### DISCUSSION THREAD

**Discussion by spike-xiong** [osmosis-labs/osmosis-frontend#1335] [Q&A] [answered]
I followed the contribution section in Readme, and my env is win10+node v16.17.1.
After I run 'yarn build', the result returned:

> lerna info run Ran npm script 'build' in '@osmosis-labs/math' in 3.0s:
$ tsc
lerna info run Ran npm script 'build' in '@osmosis-labs/pools' in 2.7s:
$ tsc
lerna ERR! yarn run build exited 1 in '@osmosis-labs/stores'
lerna ERR! yarn run build stdout:
$ tsc && yarn build:proto
$ mkdir -p ./build/account/msg/proto/generated && cp ./src/account/msg/proto/generated/* ./build/account/msg/proto/generated
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run build stderr:
The syntax of the command is incorrect.
error Command failed with exit code 1.
error Command failed with exit code 1.
lerna ERR! yarn run build exited 1 in '@osmosis-labs/stores'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by jonator** [comment, marked as answer]
Are you using a windows machine?? A build step in the stores package may require a UNIX-like shell

**Reply by spike-xiong** [reply]
Yes, so that's the reason then. The Readme might need one more requirement condition about the system.
